collector/kernel/bpf_handler.h

```diff
@@ -97,4 +101,5 @@ class BPFHandler {
   FileDescriptor &bpf_dump_file_;
   logging::Logger &log_;
   u64 last_lost_count_;
+  HostInfo const &host_info_;
```

Comment:
I don't believe that the host info will change. Can we keep a copy here instead of a reference?

Reply:
OK